tools/lkl/scripts/dpdk-sdk-build.sh

```diff
@@ -2,7 +2,8 @@
 
 dpdk_version="17.02"
 
-git clone -b v${dpdk_version} git://dpdk.org/dpdk dpdk-${dpdk_version}
+#git clone -b v${dpdk_version} git://dpdk.org/dpdk dpdk-${dpdk_version}
```

Comment:
Hajime, thank you for this comment. Your solution with git is very nice. I totally agree with this.